src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogStorageFactory.java

```diff
@@ -86,35 +86,23 @@ private LogStorage forExec(@NonNull Queue.Executable exec) {
         if (exec instanceof Run<?, ?> run && run.getAction(MonitoringAction.class) != null) {
             // it's a pipeline with monitoring data
             logger.log(Level.FINEST, () -> "forExec(" + run + ")");
-            ret = new OtelLogStorage(run, getOtelTraceService(), tracer);
+            ret = new OtelLogStorage(run, otelTraceService, tracer);
         }
         return ret;
     }
 
-    /**
-     * Workaround dependency injection problem. @Inject doesn't work here
-     */
-    @NonNull
-    private JenkinsControllerOpenTelemetry getJenkinsControllerOpenTelemetry() {
-        if (jenkinsControllerOpenTelemetry == null) {
-            jenkinsControllerOpenTelemetry = JenkinsControllerOpenTelemetry.get();
+    @Extension()
+    @Symbol("opentelemetry")
+    public static final class DescriptorImpl extends LogStorageFactoryDescriptor<OtelLogStorageFactory> {
+        @NonNull
+        @Override
+        public String getDisplayName() {
+            return "Open Telemetry";
```

Comment:
Displayed in the new "Pipeline logging" global UI configuration, see https://github.com/jenkinsci/workflow-api-plugin/pull/417

Reply:
I think this is always one word "OpenTelemetry": https://opentelemetry.io/.

```suggestion
            return "OpenTelemetry";
```